Issue 2153

Naaaaiix opened the pull request:
Fixes #2153 
Add new method `getColorRGBAParts`
Add tests for `getColorRGBAParts`
Update colours for dark on SC
Update tests

Naaaaiix committed:
Fix #2153
Create new method getColorRGBAParts
Create tests for getColorRGBAParts
Update dark colours on SC
Remove innecessary components on Toolbar

Naaaaiix committed:
Merge branch 'master' into issue-2153

Naaaaiix committed:
Update tests
Fix minor bugs

Naaaaiix committed:
Fix minor issue

Alejandro9kgm reviewed (commented):
The same bug happens if you use a button

https://user-images.githubusercontent.com/58038966/133962427-262ac7ac-5efb-45f9-9c73-d6f3f5f000b1.mp4

elzadj reviewed (changes requested):
> The same bug happens if you use a button

I confirm, the error for the Text Block is:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'p')
    at getTypographyFromSC (index.js:104936)
    at getCustomFormatValue (index.js:122495)
    at getValue (index.js:98153)
    at index.js:98323
    at renderWithHooks (react-dom.js:14938)
    at mountIndeterminateComponent (react-dom.js:17617)
    at beginWork (react-dom.js:18731)
    at HTMLUnknownElement.callCallback (react-dom.js:182)
    at Object.invokeGuardedCallbackDev (react-dom.js:231)
    at invokeGuardedCallback (react-dom.js:286)
```

and for the Button Block:

```
getTypographyFromSC.js:16 Uncaught TypeError: Cannot read properties of undefined (reading 'button')
    at getTypographyFromSC (getTypographyFromSC.js:16)
    at getCustomFormatValue (getCustomFormatValue.js:84)
    at getValue (index.js:454)
    at index.js:635
    at renderWithHooks (react-dom.js?ver=16.13.1:14938)
    at mountIndeterminateComponent (react-dom.js?ver=16.13.1:17617)
    at beginWork (react-dom.js?ver=16.13.1:18731)
    at HTMLUnknownElement.callCallback (react-dom.js?ver=16.13.1:182)
    at Object.invokeGuardedCallbackDev (react-dom.js?ver=16.13.1:231)
    at invokeGuardedCallback (react-dom.js?ver=16.13.1:286)
```

Naaaaiix committed:
Fix minor bug

Naaaaiix committed:
Fix minor bug

Naaaaiix committed:
ColorControl refactor (#2162)

* Refactor ColorControl
Clean ColorControl instances
Enhance getColorRGBAParts method
Fix TextShadowControl
Update tests

* Fix SC and palette colours

* Fix SC and palettes colour - 2

* Fix minor bug

Naaaaiix committed:
Merge branch 'master' into issue-2153
